docs/installation/linux/centos.md

```diff
@@ -139,54 +177,63 @@ makes the ownership of the Unix socket read/writable by the `docker` group.
 
 To create the `docker` group and add your user:
 
-1. Log into Centos as a user with `sudo` privileges.
+1. Log into your machine as a user with `sudo` or `root` privileges.
```

Comment:
nit: you use sudo in all the commands below and sudo is very standard, so why not just say "with sudo privileges"?